docs/build.gradle

```diff
@@ -5,9 +5,24 @@ plugins {
 def pythonExe = osWin ? "python.exe" : "python3"
 def docsVenv = "$buildDir/sphinx"
 def docsDir = projectDir
-def venvPath = (osWin ? "${docsVenv}/Scipts" : "${docsVenv}/bin")
-
+def venvPath = (osWin ? "${docsVenv}/Scripts" : "${docsVenv}/bin")
+def noDocs = project.findProperty("no.docs") ?: false
+def havePython = checkPython(pythonExe)
 
+def checkPython(executableStr) {
+    def result = exec {
+        ignoreExitValue = true
+        if (osWin) {
+            executable "cmd"
+            args += "/c"
+        } else {
+            executable executableStr
+        }
+        args += "-c"
+        args += "import sys; sys.exit(0)"
+    }
+    return result.getExitValue() == 0
+}
```

Comment:
```suggestion
def checkPython(pythonStr) {
    def result = exec {
        ignoreExitValue = true
        if (osWin) {
            executable "cmd"
            args += "/c"
            args += pythonStr
        } else {
            executable pythonStr
        }
        args += "-c"
        args += "import sys; print('checking for python'); sys.exit(0)"
    }
    return result.getExitValue() == 0
}
```

Reply:
I was seeing error in the output

-c is not a valid command; added pythonStr command to args , consistent with unix.

Reply:
Oops, thanks.